enterprise/server/cmd/ci_runner/bundle/BUILD

```diff
@@ -0,0 +1,39 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+# gazelle:default_visibility //enterprise:__subpackages__
+package(
+    default_visibility = [
+        "//enterprise:__subpackages__",
+    ],
+)
+
+genrule(
+    name = "embedsrcs_compatible_ci_runner",
+    srcs = ["//enterprise/server/cmd/ci_runner"],
+    outs = ["buildbuddy_ci_runner"],
+    cmd_bash = "cp $(SRCS) $@",
+)
+
+genrule(
+    name = "empty_for_embedsrcs",
+    outs = ["empty"],
+    cmd_bash = "touch $@",
+)
+
+# gazelle:ignore
+go_library(
+    name = "bundle",
+    srcs = ["bundle.go"],
+    embedsrcs = select({
+        "//:fastbuild": [
+            ":empty",  # embed fails with no embedsrcs
+        ],
```

Comment:
I think we should embed the CI runner unconditionally, otherwise we'd have to build with `-c opt` to test workflows locally (IIUC?)